modules/nested-tabs-html/widgets/nested-tabs-html.php

```diff
@@ -399,7 +399,7 @@ protected function register_controls() {
 			}
 
 			$dropdown_options[ $breakpoint_key ] = sprintf(
-				/* translators: 1: Breakpoint label, 2: `>` character, 3: Breakpoint value. */
```

Comment:
please revert, this is our convention.

Reply:
@rami-elementor consider it done. Thanks for pointing this out.

Reply:
Not reverted?

Reply:
Yes it's been reverted. PR seems stuck, I'll check again after pushing other changes to this file. 
![Uploading image.png…]()